### PR TITLE
PMM-8307 upgrade nginx and move nginx role to update.yml

### PR DESCRIPTION
**Pull request by nikita-b**
https://github.com/Percona-Lab/pmm-submodules/pull/1872

### CODE REVIEW

File: ansible/playbook/tasks/roles/nginx/files/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    access_log  /dev/stdout  main;
+
+    sendfile        on;
+
+    keepalive_timeout  65;
+
+
+    include /etc/nginx/conf.d/*.conf;
+}
```
Comment (atymchuk) [inline ansible/playbook/tasks/roles/nginx/files/nginx.conf:32]: Need an empty line here

**Comment by BupycHuk** [comment]
I hope this PR won't overwrite our customers configs, am I right?

**Comment by nikita-b** [comment]
> I hope this PR won't overwrite our customers configs, am I right?

@BupycHuk As I understand we don't guarantee immutability nginx.conf even in the current implementation.
1. Customer can just use the new version of container => config will be lost
2. Any package update => config will be lost. (We have active nginx repository repo in a container)

Maybe it has the sense to create a backup of a file but still, file can be lost until we'll store it in `/srv/`

**Review by BupycHuk — DISMISSED** [review]

File: ansible/playbook/tasks/roles/nginx/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        name: nginx
+        description: nginx repo
+        baseurl: http://nginx.org/packages/centos/7/$basearch/
+        gpgcheck: no
+        enabled: yes
+
+    - name: Install nginx rpm          | Install nginx rpm
+      yum:
+        name:
+          - nginx-1.20.1
```
Comment (atymchuk) [inline ansible/playbook/tasks/roles/nginx/tasks/main.yml:13]: Remember, we had a plan on preventing yum from upgrading nginx to the latest version?

File: ansible/playbook/tasks/roles/nginx/files/conf.d/pmm-ssl.conf [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ssl_prefer_server_ciphers on;
+
+ssl_session_cache shared:SSL:10m;
+ssl_session_tickets off;
+ssl_session_timeout 5m;
+
+ssl_stapling on;
+ssl_stapling_verify on;
+
+resolver 8.8.8.8 8.8.4.4 valid=300s;
```
Comment (atymchuk) [inline ansible/playbook/tasks/roles/nginx/files/conf.d/pmm-ssl.conf:14]: This should not be part of an ssl config. Let's move to nginx.conf

File: ansible/playbook/tasks/roles/nginx/files/nginx.conf [hunk 4]
```diff
@@ -0,0 +1,32 @@
+user  nginx;
+worker_processes  2;
+
+
+daemon off;
+
+error_log  stderr warn;
```
Comment (atymchuk) [inline ansible/playbook/tasks/roles/nginx/files/nginx.conf:7]: /dev/stderr is more canonical, let's use it for those less familiar with nginx

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: ansible/playbook/tasks/roles/nginx/tasks/main.yml (hunk 2 above)
Comment (atymchuk) [reply ansible/playbook/tasks/roles/nginx/tasks/main.yml:13]: Also, are these redundant now? https://github.com/percona/pmm-update/pull/134/files#diff-0180ac272b00bc7b2bb15d3b9b054cb60171c3a7022f057caa0136edad5295d3L127-L128

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: ansible/playbook/tasks/roles/nginx/tasks/main.yml (hunk 2 above)
Comment (nikita-b) [reply ansible/playbook/tasks/roles/nginx/tasks/main.yml:13]: We planned but I'm afraid we can't. We actually don't use the nginx repo at all. We're getting the nginx packages from `epel` repo and I decided just disable the nginx repo.

**Review by atymchuk — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Pull request merged**